schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+	          "items": {
+               "type": "array",
+	              "items" : {
+                    "type": "number",
+                    "minItems": 3,
+                    "maxItems": 3
+		                }
+	                 }
+	     },
+       "SUN_POSITION" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+	     },
+       "SUN_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		        }
+	         }
+	     },
+
+       "ORIENTATION" : {
+           "description": "Orientation of the sensor provided in quaternions",
+           "type": "array",
+           "items": {
+               "type": "array",
+               "items" : {
+                  "type": "number",
+                  "minItems": 4 ,
+                  "maxItems": 4
+           }
+          }
+      },
+
+       "DETECTOR_LINE_RATE": {
+           "description": "The line rates of the detector map with the associated start line and time",
+           "type": "array",
+              "items": {
+	               "type": "object",
+	                "properties": {
+	                   "START_LINE": {
+		                     "type": "number"
+	                    },
+                      "LINE_START_TIME": {
+ 	                        "type": "number"
+	                     },
+	                     "EXPOSURE_TIME": {
+		                       "type": "number"
+	                     },
+	          "required": [ "EXPOSURE_TIME"]
+	                 }
+	              }
+      },
+       "DETECTOR_SAMPLE_SUMMING": {
+           "description": "The summing in the sample-direction",
+           "type": "number"
+       },
+       "DETECTOR_LINE_SUMMING": {
+           "description": "The summing in the line-direction",
+           "type": "number"
+       },
+       "APPLY_LIGHT_TIME_CORRECTION": {
+           "description": "A flag to indcate whether the light-time correction should be applied",
+           "type": "bool"
+       },
+       "NUMBER_OF_EPHEM": {
+           "description": "The number of emphemeris points",
+           "type": "number"
+       },
+       "NUMBER_OF_QUATERNIONS": {
+           "description": "The number of quaternions",
+           "type": "number"
+       },
+       "DT_EPHEM": {
+           "description": "The time between each ephemeris point.",
+           "type": "number"
+       },
+       "T0_EPHEM": {
+           "description": "First ephemeris time - center ephemeris time",
```

Comment:
I think this should be first quaternion ephemeris time - center image time

Reply:
:+1: